packages/flutter/lib/src/material/page.dart

```diff
@@ -5,12 +5,19 @@
 import 'dart:async';
 
 import 'package:flutter/widgets.dart';
+import 'material.dart';
+import 'theme.dart';
 
 final FractionalOffsetTween _kMaterialPageTransitionTween = new FractionalOffsetTween(
```

Comment:
all the mentions of "Material" here should be "Android". Both the iOS and Android transitions are "material", they're just adapted to the local host platform.